### ISSUE TITLE
Dropout Feature

### ISSUE THREAD

**Issue by mscengsoft** [CodingTrain/Toy-Neural-Network-JS#130]
Hello everyone! I really like this library, and the Coding Train vídeos really opened my mind to all the possibilities of ANN. 

I was reading about ways to avoid overfitting and Dropout comes up a lot. Did anyone successfully implement dropout to the library? If not, could you please provide any clues on where to start?

Thank you very much :)

**Comment by Tombstone01** [comment]
My guess would be that, this neural network is very simple. Adding things like dropout, would overly complicate things, especially for people just starting out with NN. Though don't let this comment stop you from reaching for the stars. :heart: :smile:
